COMETwebapp.Tests/IterationServiceTest/IterationServiceTest.cs

```diff
@@ -0,0 +1,170 @@
+            {
+                Owner = domainOfExpertise,
+                ParameterType = simpleQuantityKind
+            };
+
+            var parameter2 = new Parameter(Guid.NewGuid(), cache, uri)
+            {
+                Owner = domainOfExpertise,
+                ParameterType = simpleQuantityKind2
+            };
+
+            var parameterValueset_1 = new ParameterValueSet()
+            {
+                ActualOption = option_A,
+                Iid = Guid.NewGuid()
+            };
+
+            var parameterValueset_2 = new ParameterValueSet()
+            {
+                ActualOption = option_A,
+                Iid = Guid.NewGuid()
+            };
+
+            var values_1 = new List<string> { "2" };
+            var values_2 = new List<string> { "3" };
+            var publishedValues = new List<string> { "123" };
+
+            this.iteration.Option.Add(option_A);
+            this.iteration.DefaultOption = option_A;
+
+            parameterValueset_1.Manual = new CDP4Common.Types.ValueArray<string>(values_1);
+            parameterValueset_1.Reference = new CDP4Common.Types.ValueArray<string>(values_1);
+            parameterValueset_1.Computed = new CDP4Common.Types.ValueArray<string>(values_1);
+            parameterValueset_1.Formula = new CDP4Common.Types.ValueArray<string>(values_1);
+            parameterValueset_1.Published = new CDP4Common.Types.ValueArray<string>(publishedValues);
+            parameterValueset_1.ValueSwitch = ParameterSwitchKind.MANUAL;
+
+            parameterValueset_2.Manual = new CDP4Common.Types.ValueArray<string>(values_2);
+            parameterValueset_2.Reference = new CDP4Common.Types.ValueArray<string>(values_2);
+            parameterValueset_2.Computed = new CDP4Common.Types.ValueArray<string>(values_2);
+            parameterValueset_2.Formula = new CDP4Common.Types.ValueArray<string>(values_2);
+            parameterValueset_2.Published = new CDP4Common.Types.ValueArray<string>(publishedValues);
+            parameterValueset_2.ValueSwitch = ParameterSwitchKind.MANUAL;
+
+
+            parameter.ValueSet.Add(parameterValueset_1);
+            parameter.ValueSet.Add(parameterValueset_2);
+
+            elementDefinition_1.Parameter.Add(parameter);
+            elementDefinition_1.ContainedElement.Add(elementUsage_1);
+            elementDefinition_1.ContainedElement.Add(elementUsage_2);
+
+            elementDefinition_2.Parameter.Add(parameter);
+            elementDefinition_2.Parameter.Add(parameter2);
+
+            this.iteration.Element.Add(elementDefinition_1);
+            this.iteration.Element.Add(elementDefinition_2);
+            this.iteration.TopElement = elementDefinition_1;
+
+            this.iterationService = new IterationService(this.iteration);
+        }
+
+        [Test]
+        public void VerifyGetParameterValueSets()
+        {
+            Assert.IsNotEmpty(iterationService.GetParameterValueSets());
```

Comment:
we typically make use of the `Assert.That` syntax, I think this then becomes:

`Assert.That(iterationService.GetParameterValueSets(), Is.Not.Empty);`